vendor/github.com/ethersphere/go-metrics/resetting_timer.go

```diff
@@ -0,0 +1,237 @@
+package metrics
```

Comment:
This is the main change in this PR. Since `govendor` does not include tests, you can find them at https://github.com/ethersphere/go-metrics/blob/master/resetting_timer_test.go